requirements.txt

```diff
@@ -1,18 +1,199 @@
-amqp==2.2.2
```

Comment:
What was the name of the package you installed? Right now, all of the python packages on you computer are in the requirements.txt. That's a bit too much ;-)

 If you send me the name of the package, I will send you a clean `requirement.txt` file to update this one before merging.